scripts/update/update-git.sh

```diff
@@ -1,11 +1,9 @@
-echo Update all repositories ...
-cd ../../../
+echo Update files...
+cd ../..
 for D in */; do
 	echo --------------
 	echo "$D";
 	cd $D
-	git pull
```

Comment:
I think this is wrong, the ``update-git.sh`` should pull all new changes?

Reply:
`git sync`?

Reply:
I will use it